### PR TITLE
CC-4105 enforce md5 auth for zookeeper quorum

### PR DESCRIPTION
**Pull request by ssOlexBaiko**
[Jira](https://jira.zenoss.com/browse/CC-4105)

### CODE REVIEW

**Comment by ssOlexBaiko** [comment]
retest this

File: start-zookeeper [hunk 1]
```diff
@@ -12,12 +12,42 @@ fi
...
     echo $ZKID > $ZK_DATA/myid
 fi
+
+if [ -z "$ZK_USERNAME" ]; then
+    ZK_USERNAME="ccuser"
+fi
+
+if [ -z "$ZK_PASSWD" ]; then
+    ZK_PASSWD="zenoss"
+fi
```
Comment (smousa) [inline start-zookeeper]: this should be like line 5

**Comment by jplouis** [comment]
retest this

**Comment by ssOlexBaiko** [comment]
retest this

**Comment by jplouis** [comment]
retest this

**Comment by jplouis** [comment]
retest this

**Pull request merged**